PerimeterXModule/Internals/PxContext.cs

```diff
@@ -46,7 +46,7 @@ public PxContext(HttpContext context, PxModuleConfigurationSection pxConfigurati
             {
                 if (Array.IndexOf(PxConstants.PX_COOKIES_PREFIX, key) > -1)
                 {
-                    PxCookies.Add(key, contextCookie.Get(key).Value);
+					PxCookies[key] = contextCookie.Get(key).Value;
```

Comment:
1. Following this approach is to actually take the last cookie of the same name and not the first one. Is that's the original intention?
2. Please convert the tabs to spaces with the auto-formatting (ctrl+shift+f) or manually, like in the other PR...